.jenkins/pytorch/test.sh

```diff
@@ -33,6 +30,20 @@ if [ -n "${IN_CIRCLECI}" ]; then
   fi
   if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
     export PYTORCH_COLLECT_COVERAGE=1
+    # TODO move this to docker
+    pip_install unittest-xml-reporting coverage
+    # install lcov with 1.15 (gcc 9 requires lcov>1.14-2, apt-get will only install 1.12)
+    # outside of workplace, so we don't need to clean it up
+    pushd ..
+    wget https://github.com/linux-test-project/lcov/releases/download/v1.15/lcov-1.15.tar.gz
+    tar -xf lcov-1.15.tar.gz
+    pushd lcov-1.15
+    sudo make install
+    popd
+    popd
```

Comment:
Since lcov is just the Perl script, wouldn't it be easier to just download https://raw.githubusercontent.com/linux-test-project/lcov/master/bin/lcov into `~/local/bin`